### ISSUE TITLE
[FEATURE_REQUEST] Multiple Search Engines Simultaneously

### ISSUE THREAD

**Issue by shimizurei** [Lissy93/dashy#206]
### Is your feature request related to a problem? If so, please describe.

Thank you for adding the search function! I was hoping it could be made more multifunctional. With just some text, you could change what search engine you use on the fly. Plus, the user could expand it to encapsulate as many sites as they wish.

### Describe the solution you'd like

Here's an example of the idea:

#### Config File

```
"appConfig":
  "webSearch": {
    "searchEngine": "multi",
    "multiSearchEngine": {
         "/g": "google",
         "/d": "duckduckgo",
         "/rd": "reddit",
         "/yt": "youtube",
         "/sp": "https://open.spotify.com/search/",
         "/wk": "wikipedia",
         "/md": "https://mangadex.org/titles?q=",
         "/js": "http://jisho.org/search/",
    }
 }
```

That way the user can add their own engines based on sites they frequently/commonly use.

You could also use bangs (!) if it works better: e.g. "!d": "duckduckgo"

#### Examples of Use

- /rd "jjk movie discussion"
- /js "終了"
- /yt "how to hold a cat"
- /md "naruto"

### Priority

Medium (Would be very useful)

### Is this something you would be keen to implement

No

**Comment by Lissy93** [comment]
Love the idea! I had not thought of this, but it makes sense, and should be pretty easy to implement.
I'll get it done this weekend, and update the ticket when it's released :)

**Comment by Lissy93** [comment]
Heya, this is implemented in #224 :)

Once you've updated to V 1.7.6, you can use it with something like:

```yaml
appConfig:
  webSearch:
    searchEngine: 'duckduckgo'
    openingMethod: 'newtab'
    searchBangs:
      /r: reddit
      /w: wikipedia
      /s: https://whoogle.local/search?q=
      /a: https://www.amazon.co.uk/s?k=
      ':wolf': wolframalpha
      ':so': stackoverflow
      ':git': github
```

Hope that helps :)
